app/screens/webview/webview.tsx

```diff
@@ -86,7 +86,6 @@ export const WebViewScreen: React.FC<Props> = ({ route }) => {
           sendPayment: async (paymentRequestStr) => {
             navigate("sendBitcoinDestination", {
               payment: paymentRequestStr,
-              autoValidate: true,
```

Comment:
Why is that been removed?

Reply:
Were using it for blink fiat to skip showing the invoice this is confusing for the user

Reply:
It was breaking the linking into the mobile application because autoValidate was not present so it was not receiving any invoice at all. Where is autoValidate supposed to be false? I only see it as true.

Reply:
default = undefined = false, so for every use case where it's not explicitly true

Reply:
Alright let me adjust.

Reply:
The way the logic is set up currently, however, means that if autoValidate isn't present, no invoice is being passed in at all.

Reply:
> Were using it for blink fiat to skip showing the invoice this is confusing for the user

Actually the destination is now present on the confirmation screen. Do you still want to show it on the destination screen as well?